src/Row.js

```diff
@@ -52,7 +52,7 @@ const Row = (props) => {
 
   const classes = mapToCssModules(classNames(
     className,
-    noGutters ? 'no-gutters' : null,
+    noGutters ? 'gx-0' : null,
```

Comment:
I wonder if we should convert this to a convenience prop for all gutter sizes.  Alternatively could remove the prop entirely and leave class handling in user land.  Thoughts?

Reply:
I think it's nice to avoid breaking changes if possible. Perhaps we can keep the prop for now, but add a deprecation warning? Any more fine tined adjustments to the gutter can still be achieved through normal classes.

Reply:
Sure let's add the warning.  Thanks!

Reply:
👍